app/src/components/Settings/index.tsx

```diff
@@ -99,11 +45,7 @@ const Settings = () => {
   })
 
   return (
-    <form
-      data-testid={'settings-form'}
-      onSubmit={formik.handleSubmit}
-      className={SETTINGS_STYLES.FORM}
-    >
+    <form data-testid={'settings-form'} onSubmit={formik.handleSubmit} className={STYLES.FORM}>
       <InputNumber
```

Comment:
I think all this components could be extracted to the `components.tsx` file.

So instead of having `InputNumber`, we have a component `InputPomodoroDuration` that has all these lines inside.

Do the same with the other ones.